### PR TITLE
Getting started documentation

### PR DESCRIPTION
**Pull request by DmitriyNoa**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Getting started
+
+## Dependencies:
+### Minimal:
+1. NodeJS version >=10.0.0 <12.0.0.
+2. [Yarn](https://yarnpkg.com/lang/en/).
+
+### Full:
```
Comment (arjunu) [inline docs/getting-started.md]: should we mention what full install is for? Like what docker and Ganache is required for?

**Comment by arjunu** [comment]
Root file [README](`https://github.com/Neufund/platform-frontend/blob/master/README.md`) already has some of these steps. Will you merge these files?

File: docs/getting-started.md (hunk 1 above)
Comment (DmitriyNoa) [reply docs/getting-started.md]: Yes, I will mention this as soon as I find out how to run everything locally with local backend

Comment (drakkein) [reply docs/getting-started.md]: Also it's worth to mention that we have ability to run frontend on remote backend instance `start:remote:vm` task, which requires to have `NF_VM_ADDRESS="ip here"` set in `.env` file.

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Setting environment.
+1. Clone the platform-frontend repo:
+```git clone git@github.com:Neufund/platform-frontend.git```.
+2. cd into the project:
+```cd platform-frontend```.
+3. Install dependencies ````yarn````.
+4. Generate contract artifacts ````yarn contract-artifacts:generate````.
+5. Setup environment variables:
+- Make sure that .env file is in place ./packages/web/.env
+- If it's not create it ````touch .env````.
```
Comment (drakkein) [inline docs/getting-started.md]: Also `.env` should be created automatically from `.env.example` while running any `start *` task

**Comment by drakkein** [comment]
@DmitriyNoa If you don't have it yet, you can ask @pstroinski about remote VM to run backend on it

File: docs/getting-started.md (hunk 2 above)
Comment (arjunu) [reply docs/getting-started.md]: Yeah I don't rem creating it myself

File: docs/getting-started.md (hunk 1 above)
Comment (DmitriyNoa) [reply docs/getting-started.md]: Yes, but it generates it without those 2 envs variables provided and fails immediately, so you need to stop the process, define vars and start again

File: docs/getting-started.md (hunk 2 above)
Comment (DmitriyNoa) [reply docs/getting-started.md]: Yes, but it generates it without those 2 envs variables provided and fails immediately, so you need to stop the process, define vars and start again

**Comment by DmitriyNoa** [comment]
I moved the getting started docs to the main README in the web package. The file already contained some information so I moved the missing parts to it. Please check

File: packages/web/README.md [hunk 3]
```diff
@@ -86,3 +130,26 @@ yarn lint:fix
 ```
 
 job
+
+## Possible issues:
```
Comment (arjunu) [inline packages/web/README.md]: 👍

File: packages/web/README.md [hunk 4]
```diff
@@ -9,12 +9,48 @@ A web UI of neufund platform.
...
+## Setting environment.
+
+1. Clone the platform-frontend repo: `git clone git@github.com:Neufund/platform-frontend.git`.
+2. cd into the project: `cd platform-frontend`.
+3. Install dependencies `yarn`.
+4. Generate contract artifacts `yarn contract-artifacts:generate`.
+5. Setup environment variables:
+
+- Make sure that .env file is in place ./packages/web/.env.
+- If it's not create it run `yarn start:remote`.
```
Comment (arjunu) [inline packages/web/README.md]:
```suggestion
- If it's not, create it and run `yarn start:remote`.
```

File: packages/web/README.md [hunk 5]
```diff
@@ -9,12 +9,48 @@ A web UI of neufund platform.
 
 **Storybook** (latest `master`): https://neufund.github.io/platform-frontend/
 
+### Dependencies:
+
+1. NodeJS version >=10.0.0 <12.0.0.
+2. [Yarn](https://yarnpkg.com/lang/en/).
+
+## Setting environment.
```
Comment (arjunu) [inline packages/web/README.md]:
```suggestion
## Setting up the environment
```

**Comment by arjunu** [comment]
There are some sectioning issues in the file:
- Uploading contracts is under section "Running project"
", should be in a different one
- "Fix all autofixable errors" and "Production build" sections are under "Running tests", instead should be separate
- "Extracting translations from the app" and "Update contract artifacts submodule" are under the section "Running storybook" , should be in a different one
- There's a typo "job" above "Possible issues" title

**Pull request merged**